torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -391,9 +391,20 @@ def gen_check(handle_kind, idx, name, tensor):
         # inline done by the host compiler
         self.prefix.splice(
             """
+            bool _check_aoti_runtime_check_inputs_env() {
+                static bool result = [] {
```

Comment:
Do we have to use a lambda here?

Reply:
I think as long as you declare `result` as static, its initialization will only be done once.

Reply:
@desertfire Changed to not using lambda now.